packages/dds/tree/src/core/forest/editableForest.ts

```diff
@@ -51,10 +54,19 @@ export function initializeForest(
 	content: readonly ITreeCursorSynchronous[],
 	revisionTagCodec: RevisionTagCodec,
 	idCompressor: IIdCompressor,
+	visitAnchors = false,
 ): void {
 	assert(forest.isEmpty, 0x747 /* forest must be empty */);
 	const delta: DeltaRoot = deltaForRootInitialization(content);
-	applyDelta(delta, forest, makeDetachedFieldIndex("init", revisionTagCodec, idCompressor));
```

Comment:
@Abe27342 FYI all that `applyDelta` does is 
1. acquire a visitor from the forest
2. call `visitDelta` 
3. free the visitor

So we're still doing the exact same thing in the case that `visitAnchors` is false.